### PR TITLE
Change drc-hash test to assert_equal to whole hash instead of just ch…

### PR DESCRIPTION
**Pull request by mayerm94**
Assert whole hash value in drc test so that we have failing tests if somebody changes the way that the hashes are generated.

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: NA
- [x] JIRA ticket links to this PR NA
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: NA
- [x] Justification for using JIRA tests: NA
- [x] JIRA tests have been added to sprint NA

**Reviewer 1:**

Name: @losborne
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] N/A JIRA tests have been run and pass
- [x] N/A You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: test/unit/cql_loader_test.rb [hunk 1]
```diff
@@ -38,8 +38,9 @@ class CQLLoaderTest < ActiveSupport::TestCase
       assert_equal 1, CqlMeasure.all.count
       measure = CqlMeasure.all.first
       # Confirm that the source data criteria with the direct reference code starts with 'drc-'
```
Comment (lukoktonos) [inline test/unit/cql_loader_test.rb]: I would replace this comment with one explaining that if the new lines are failing, the way the codes are generated has been changed, which needs some sort of update rake task if intended.

**Review by lukoktonos — CHANGES_REQUESTED** [review]

**Review by lukoktonos — APPROVED** [review]
